src/site/layouts/page-breadcrumbs.html

```diff
@@ -12,6 +12,25 @@
   {% include "src/site/includes/decision-reviews-banner.html" %}
 {% endif %}
 
+{% if banner %}
+  <div class="usa-alert-full-width usa-alert-full-width-warning" role="region">
+    {% assign bannerType = 'warning' %}
+    {% if banner.type %}
+      {% assign bannerType = banner.type %}
+    {% endif %}
+    <div class="usa-alert usa-alert-{{ bannerType }}" aria-live="assertive" role="alert">
+      <div class="usa-alert-body">
+        {% if banner.title %}
+          <h3 class="usa-alert-heading">{{ banner.title }}</h3>
```

Comment:
I'd like to propose adding an `ID` here so we can reference it on line 16 using `aria-labelledby`. 

@aubreyarcangel and I worked on the one-off banner in place, and we found that adding these attributes gave the banner a descriptive name in screen readers' landmarks menu, which was a nice boost for assistive devices.

```html
 <h3 class="usa-alert-heading" id="alert-heading-text">{{ banner.title }}</h3>
```